.github/workflows/publish.yml

```diff
@@ -22,7 +22,7 @@ jobs:
 
       - name: Version
         env:
-          PR_TITLE: ${{ toJson(github.event.pull_request.title) }}
+          PR_TITLE: ${{ toJson(github.event.head_commit.message) }}
```

Comment:
This is for publishing, when the PR merged and the message should be determined from the triggering commit (the merge)